patches/chrome-common-media-cdm_host_file_path.cc.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/common/media/cdm_host_file_path.cc b/chrome/common/media/cdm_host_file_path.cc
+index a1624dfbd226cf1c664187853a89f9531c8a777e..31e8a3200359ca0f55898be318f9de42fc487a66 100644
+--- a/chrome/common/media/cdm_host_file_path.cc
++++ b/chrome/common/media/cdm_host_file_path.cc
+@@ -44,7 +44,11 @@ void AddCdmHostFilePaths(
+ #if defined(OS_WIN)
+ 
+   static const base::FilePath::CharType* const kUnversionedFiles[] = {
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
why is there a patch for this if the entire file is effectively removed by this? https://github.com/brave/brave-core/pull/2158/files#diff-b32e3750ef0eab90392d30f5913f0d60R7